modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/QueueTable.scala

```diff
@@ -0,0 +1,533 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
```

Comment:
You should probably change this in your editor. 😉

Reply:
We should change on the build I think